dashboard/app/helpers/notes_helper.rb

```diff
@@ -0,0 +1,20 @@
+module NotesHelper
+  include LocaleHelper
+
+  def get_slides_by_video_key(video_key)
+    slides = try_t('slides.' + video_key)
```

Comment:
Is it worth extracting `'slides.' + video_key` to a variable (it is used in `slides_english` also)?

Prefer string interpolation, e.g., "slides.#{video_key}".